ProjectCoimbra.UWP/Project.Coimbra/Helpers/SongPagesHelper.cs

```diff
@@ -185,5 +184,31 @@ public static async Task FillListBoxAsync(ListBox listBox, string selectedValue,
                 nextButton.IsEnabled = true;
             }
         }
+
+        /// <summary>
+        /// Removes Item from list
+        /// </summary>
+        /// <param name="key">item key</param>
+        public static async Task RemoveFile(string key)
```

Comment:
Please call this method `RemoveFileAsync` as per convention so that callers know that the method is async and needs to be called with `await`